src/root.component.tsx

```diff
@@ -11,6 +11,7 @@ export default function Root(props: RootProps) {
   const [isLoggedIn, setIsLoggedIn] = useState(true);
   const [userName, setUserName] = useState(null);
   const [locale, setLocale] = useState("en");
+  const [navBarConfig, setNavBarConfig] = useState({ url: "" });
```

Comment:
good work @PremnishaTW. Interesting how you manage configs, you might also want to checkout [openmrs-esm-module-config](https://github.com/openmrs/openmrs-esm-module-config), an esm module that we use to manage configs it lets you define a config schema and pass configs at either runtime and build time  :-) cc @brandones

Reply:
Thank you @fatmali we currently have our config in a custom module. Migration to the openmrs-esm-module-config is on our roadmap we will be doing that.